config/jobs/ppc64le-cloud/build-docker/postsubmit-test-repo-docker.yaml

```diff
@@ -0,0 +1,129 @@
+postsubmits:
+ ppc64le-cloud/docker-ce-build:
+    - name: postsubmit-test-docker-staging
+      cluster: k8s-ppc64le-cluster
+      decorate: true
+      run_if_changed: 'env/test-staging.list'
+      branches:
+        - main
+      reporter_config:
+        slack:
+          channel: 'prow-job-notifications'
+          job_states_to_report:
+           - success
+           - failure
+           - error
+          report_template: 'Job {{.Spec.Job}} of type {{.Spec.Type}} ended with state {{.Status.State}}. <!subteam^S02N6DWBX0F> <{{.Status.URL}}|View logs>'
+      spec:
+        nodeSelector:
+          dedicated: runtimesTeam
+        tolerations:
+        - key: dedicated
+          operator: "Equal"
+          value: runtimesTeam
+          effect: "NoSchedule"
+        containers:
+        - image: quay.io/powercloud/docker-ce-build
+          command:
+          - /bin/bash
+          args:
+          - -c
+          - |
+            set -o errexit
+            set -o nounset
+            set -o pipefail
+            set -o xtrace
+
+            #Timestamp file from the tracking branch (required to access the proper s3 directory)
+            TIMESTAMP_FILE=postsubmit-build-docker
```

Comment:
I think these jobs are for testing the binaries from the Docker community's stagging and release websites.
But the comment here says 
`required to access the proper s3 directory` can you please clarify why exactly Timestamp file is needed.
Also the need for fetching `S3_SECRET_AUTH` from `docker-s3-credentials` below?

Reply:
Even if the packages are installed from the docker's websites, we still need to access date from our internal s3 directory,  In particular we need to know what distribution were built and do the test for all of them

Reply:
IMO, files `env/test-staging.list` and `env/test-release.list` should be having the details of date, and distribution details that are to be tested from docker's websites rather than going back to your s3 directory.

The `env/test-release.list` can have a fixed list of distributions that need to be tested and can have a single DATE parameter to be changed manually whenever you want to run tests for packages from the website.

If the distribution list changes in the future with a date, you may change that too when you manually update DATE.

Reply:
I am not sure if I would agree to set the list of distributions manually in the .list files. 
FYI we are populating the list of distribution automatically by looking at the directory names contained in the  [docker-ce-packaging](https://github.com/docker/docker-ce-packaging/tree/master/deb) repository maintained by Docker. 
This list of distro is evolving quite a bit, and this part the automation has been working quite smoothly so far. So I would better keep this in place. 
I have reviewed the code and I think we calculate that list of distro all the times and do not have any dependency on s3 data for that purpose. So my bad on this.
However we are storing our logs somewhere as a proof of test on the s3 directory and I would like to keep this as it is if possible.